README.md

```diff
@@ -1 +1,64 @@
-# Awesome
\ No newline at end of file
+# Awesome books
+
+Awesome books is a project that receive 2 values (a book title and its author) and save them in the local storage when you click on add button. The books saved on local storage are displayed in the HTML page, every "book" displayed has a delete button in case you want delete it.
+
+
+## Live version link:
+
+Pending...
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A browser that supports html5
+- Git 
+- A GitHub account
+
+### Install
+- Go to this repo and copy the project link
+        https://github.com/roodjinocherilus/Awesome.git
+- Open your terminal and clone the repo with this command "git clone https://github.com/roodjinocherilus/Awesome.git"
+- Now, you have a copy on your pc. 
+- To open it, make double-click on the HTML file.
+
+
+## Authors
+
+👤 **Sandro Hernandez**
+
+- GitHub: @sambeck87
+- Twitter: @sambeck4488
+- LinkedIn: (https://www.linkedin.com/in/sandro-israel-hern%C3%A1ndez-zamora-899386a4/)
+
+👤 **Roodjino Cherilus**
+
+- GitHub: @roodjinocherilus
+- Twitter: @roodjinocherilus
+- LinkedIn: https://github.com/roodjinocherilus
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- Kindly make sure to update your README file using the new template provided by Microverse as the[ last code reviewer suggested. ](https://github.com/roodjinocherilus/Awesome/pull/1/files#r1034172721)